src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -7,14 +7,1284 @@
+    pmaxThreshold1: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T3`, Watts.
+     */
+    pmaxThreshold2: types.TUint32,
+
+    /**
+     * Maximum active power threshold for tariff `T4`, Watts.
+     */
+    pmaxThreshold3: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T1`, VAR.
+     */
+    rmaxThreshold0: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T2`, VAR.
+     */
+    rmaxThreshold1: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T3`, VAR.
+     */
+    rmaxThreshold2: types.TUint32,
+
+    /**
+     * Maximum reactive power threshold for tariff `T4`, VAR.
+     */
+    rmaxThreshold3: types.TUint32,
+
+    /**
+     * Power averaging interval, in minutes (1-60).
+     */
+    tint: types.TUint8,
+
+    /**
+     * Start date of the monthly billing period.
+     */
+    calcPeriodDate: types.TUint8,
+
+    /**
+     * Display active time.
+     */
+    timeoutDisplay: types.TUint8,
+
+    /**
+     * Display active time for each screen.
+     */
+    timeoutScreen: types.TUint8,
+
+    /**
+     * Display settings for meter readings (first 32 screens).
+     */
+    displaySet1: IDisplaySet1OperatorParameter,
+
+    /**
+     * Display settings for meter readings.
+     */
+    displaySet2: IDisplaySet2OperatorParameter,
+
+    /**
+     * Display settings for meter readings.
+     */
+    displaySet3: IDisplaySet3OperatorParameter,
+
+    /**
+     * Relay settings.
+     */
+    relaySet: IRelaySetOperatorParameter,
+
+    /**
+     * Controls the data transmission speed for UART interfaces.
+     */
+    speedOptoPort: ISpeedOptoPortOperatorParameter,
+
+    /**
+     * Integration period for energy profiles.
+     * `15` - `15` minutes; `0`, `30` - `30` minutes; `60` - `60` minutes.
+     * since build 0.0.17
+     */
+    ten: types.TUint8,
+
+    /**
+     * Voltage averaging interval.
+     * `0`, `1`, `3`, `5`, `10`, `15`, `30` minutes.
+     * since build 0.0.17
+     */
+    tu: types.TUint8,
+
+    /**
+     * Interval for tracking power off events, in minutes.
+     * since build 0.0.17
+     */
+    timeIntervalPowerOff: types.TUint8,
+
+    /**
+     * Reserved byte.
+     */
+    reserved: types.TUint8,
+
+    /**
+     * Timeout for relay deactivation due to poor voltage, seconds.
+     */
+    timeoutBadVAVB: types.TUint8,
+
+    /**
+     * Maximum threshold for the frequency of the grid voltage.
+     */
+    freqMax: types.TUint8,
+
+    /**
+     * Minimum threshold for the frequency of the grid voltage.
+     */
+    freqMin: types.TUint8,
+
+    /**
+     * Year of parameters recording.
+     */
+    year: types.TUint8,
+
+    /**
+     * Month of parameters recording.
+     */
+    month: types.TUint8,
+
+    /**
+     * Date of parameters recording.
+     */
+    date: types.TUint8,
+
+    /**
+     * The number of digits after the decimal point for displaying energy values.
```

Comment:
in html generated docs this whole comment probably will be in a single line

Reply:
fixed